.github/workflows/automate-review-labels.yml

```diff
@@ -39,24 +40,35 @@
                   let text = context.payload.comment.body;
                   const regex = new RegExp(team + '[^\-]|' + team + '$');
                   let result = regex.test(text);
+                  let found_label = false;
```

Comment:
This variable needs to be set outside the loop. Otherwise, it is reset to false in each iteration.

Reply:
Is it not? I thought the loop starts later? Or am I missing something?